spec/models/project_duplicator_spec.rb

```diff
@@ -92,4 +91,64 @@
       end
     end
   end
+
+  context 'with scheduled children' do
+    # Creates a timeline and returns nodes stored in a hash.
+    let!(:nodes) { ProjectGroupFactoryHelper.create_full_timeline }
+    let(:loan) { nodes[:root].project }
+
+    # Break each of the nodes out into a let so that we can examine them individually.
+    ProjectGroupFactoryHelper::NODE_NAMES.each do |name|
+      let(name) { nodes[name] }
+    end
```

Comment:
Could you do this instead:
```
nodes.each { |k, v| let(k) { v } }
```
and then eliminate the manually synced `NODE_NAMES` constant?

Reply:
Apparently not—unless you're willing to accept `@` signs in front of the variable names, which doesn't seem all that onerous to me... 

Inside `before`:
```
 nodes.each { |k, v| instance_variable_set("@#{k}", v) }
```

Reply:
@'s are bad. The whole point of lets is that they "let" you move stuff around, up and down the scope tree.